.editorconfig

```diff
@@ -14,6 +14,87 @@ indent_size = 4
 [project.json]
 indent_size = 2
 
+# C# files
+[*.cs]
```

Comment:
nit: add `dotnet_sort_system_directives_first = true` (this doesn't match the default, but it does match the existing convention)